tests/components/tasmota/test_device_trigger.py

```diff
@@ -979,18 +992,22 @@ async def test_attach_remove_unknown1(hass, device_reg, mqtt_mock, setup_tasmota
         set(), {(dr.CONNECTION_NETWORK_MAC, mac)}
     )
 
-    remove = await async_attach_trigger(
+    remove = await async_initialize_triggers(
```

Comment:
Side note: We didn't test this before, but for the future it would be good to assert something after this call.